src/core/client/auth/local/local.graphql

```diff
@@ -6,6 +6,10 @@ enum View {
 }
 
 type Local {
+  authToken: String
+  authExp: Int
+  authJTI: String
```

Comment:
Where does this get populated?

Reply:
Here: https://github.com/coralproject/talk/blob/next/src/core/client/framework/lib/relay/localState.ts